bgpd/bgp_vty.c

```diff
@@ -260,9 +260,12 @@ bgp_vty_find_and_parse_afi_safi_vrf (struct vty *vty, struct cmd_token **argv, i
   if (*vrf == VRF_UNKNOWN)
     {
       vty_out (vty, "View/Vrf specified is unknown: %s", vrf_name);
+      *idx = 0;
```

Comment:
sorry too many threads going on, was thinking about the next comment I was going to write on this (WRT bgp = NULL).  the comment still applies, please read "why set *idx = 0 to here ..."